src/main/java/com/github/underscore/lodash/U.java

```diff
@@ -32,6 +32,9 @@
 import com.github.underscore.PredicateIndexed;
 import com.github.underscore.Tuple;
 import com.github.underscore.Optional;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
```

Comment:
We may use absolute class name for `BigDecimal` and `BigInteger`. No import needed.